pkg/pods/pod_manifest.go

```diff
@@ -28,7 +28,7 @@ type LaunchableStanza struct {
 type PodManifest struct {
 	Id                string                      `yaml:"id"` // public for yaml marshaling access. Use ID() instead.
 	LaunchableStanzas map[string]LaunchableStanza `yaml:"launchables"`
-	Config            map[string]interface{}      `yaml:"config"`
+	Config            map[interface{}]interface{} `yaml:"config"`
```

Comment:
YAML is confusing enough, us adding a new constraint on the otherwise freeform YAML `config` stanza is going to be deeply unintuitive. Agreed that we should discourage it though, it *is* weird.